rust/src/lockfile.rs

```diff
@@ -57,24 +57,29 @@ fn lockfile_parse<P: AsRef<Path>>(filename: P,) -> Fallible<LockfileConfig> {
 /// Lockfile format:
 /// ```
 /// {
-///    "packages": [
-///        [
-///          "NEVRA1",
-///          "<digest-algo>:<digest>"
-///        ],
-///        [
-///          "NEVRA2",
-///          "<digest-algo>:<digest>"
-///        ],
+///    "packages": {
+///        "name1": {
```

Comment:
This rules out multilib (e.g. having both `glibc.x86_64` and `glibc.i386` though).  Which to be fair, AFAIK we don't *currently* have a use case for (I guess we'd probably tell people to build 32-bit-only containers).

And I wouldn't be at all surprised if other parts of rpm-ostree failed in multilib scenarios (though we did try a while back to get all of the file coloring stuff right).

So...probably OK to keep as is?  But maybe worth a comment at least like:
`// Note hashing the packages by name won't work with multilib`?

Reply:
Yeah, that's a good point. I added a comment to that effect. I guess it also rules out installing multiple kernels like on traditional systems, which isn't really supported anyway right now.